src/wovnio/wovnphp/SSI.php

```diff
@@ -38,4 +38,13 @@ public static function readFileRecursive($includePath, $rootDir, &$limit) {
 
       return $code;
     }
+
+    private static function get_contents($includePath) {
+      ob_start();
+      include $includePath;
+      $contents = ob_get_contents();
```

Comment:
Also you can use `ob_get_clean()`.
This is `ob_get_contents()` + `ob_end_clean()`.
http://phpspot.net/php/man/php/function.ob-get-clean.html